### PR TITLE
docs: Add alert list document

### PR DESCRIPTION
**Pull request by zhengjianhui**
Add a list of alerts for easy subsequent maintenance and viewing

### CODE REVIEW

**Comment by homura** [comment]
It would be more clear and configurable when setting the threshold in the expression directly

```python
avg(
  histogram_quantile(
    0.90, 
    sum(rate(muta_consensus_time_cost_seconds_bucket{type="exec"}[5m])
) by (le, instance))) >= $threshold
```

File: apm/docs/alert_table.md [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+    <th>Expression</th>
+    <th>Level</th>
+    <th>Thresholds</th>
+    <th>Description</th>
+  </tr>
+</thead>
+<tbody>
+  <tr>
+    <td>TPS</td>
+    <td>avg(rate(muta_consensus_committed_tx_total[5m]))</td>
```
Comment (homura) [inline apm/docs/alert_table.md:136]: It is normal for the TPS to be 0 when the benchmark stops.

```python
# received send_transaction request count
sum (increase(muta_api_request_result_total{type="send_transaction"}[5m])) > 0 
&& 
avg(rate(muta_consensus_committed_tx_total[5m])) <= 0
```

Comment (zhengjianhui) [reply apm/docs/alert_table.md:136]: 目前按照时间来判断，不可能永远都为0

Comment (zhengjianhui) [reply apm/docs/alert_table.md:136]: Currently judging by time, it can't always be zero.

Comment (zhengjianhui) [reply apm/docs/alert_table.md:136]: With the subsequent changes, for now I want everyone to be able to see it first and then add to it together!

**Comment by zhengjianhui** [comment]
> It would be more clear and configurable when setting the threshold in the expression directly
> 
> ```python
> avg(
>   histogram_quantile(
>     0.90, 
>     sum(rate(muta_consensus_time_cost_seconds_bucket{type="exec"}[5m])
> ) by (le, instance))) >= $threshold
> ```

With the subsequent changes, for now I want everyone to be able to see it first and then add to it together!

File: apm/docs/alert_table.md (hunk 1 above)
Comment (homura) [reply apm/docs/alert_table.md:136]: When the benchmark script stops, the TPS is 0 and this triggers an alert, however, this is actually normal and none of us want the monitoring system to be the boy who cried wolf

Comment (zhengjianhui) [reply apm/docs/alert_table.md:136]: This alert can be turned off at the same time as benchmark is turned off, which also comes as a companion

**Comment by homura** [comment]
lgtm

**Pull request merged**